acta-universitatis-agriculturae-et-silviculturae-mendelianae-brunensis.csl

```diff
@@ -0,0 +1,595 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Acta Universitatis Agriculturae et Silviculturae Mendelianae Brunensis</title>
+    <title-short>Acta Univ. Agric. Silvic. Mendelianae Brun. </title-short>
+    <id>http://www.zotero.org/styles/acta-universitatis-agriculturae-et-silviculturae-mendelianae-brunensis</id>
+    <link href="http://www.zotero.org/styles/acta-universitatis-agriculturae-et-silviculturae-mendelianae-brunensis" rel="self"/>
+    <link href="http://www.zotero.org/styles/ceska-zemedelska-univerzita-v-praze-fakulta-agrobiologie-potravinovych-a-prirodnich-zdroju" rel="template"/>
+    <link href="https://acta.mendelu.cz/for-authors.html" rel="documentation"/>
+    <contributor>
+      <name>Matěj Božik</name>
+      <email>bozik@af.czu.cz</email>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="botany"/>
+    <category field="science"/>
+    <category field="zoology"/>
```

Comment:
Please add the ISSN and eISSN here, per https://github.com/citation-style-language/styles/wiki/Style-Requirements#9---issn-and-eissn. https://acta.mendelu.cz/ shows it's:

"ISSN 1211-8516 (Print)
ISSN 2464-8310 (Online)"